File: 1_two_sum/two_sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hash = {}

        for i in range(0, len(nums)):
            num = target - nums[i]

            if num in hash and hash[num] != i:
                return [i, hash[num]]
            hash[nums[i]] = i

File: 1_two_sum/two_sum.go
package twosum

func twoSum(nums []int, target int) []int {
	hash := make(map[int]int)

	for key, value := range nums {
		n := target - value

		if val, ok := hash[n]; ok {
			return []int{key, val}
		}

		hash[value] = key
	}

	panic("ERR")
}